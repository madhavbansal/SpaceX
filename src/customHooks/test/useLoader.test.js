import React from 'react';
import renderer from 'react-test-renderer';
import useLoader from '../useLoader';


jest.mock('@material-ui/core/CircularProgress', ()=> (<></>))

test('Test the useLoader Component', () => {
  const component = renderer.create(
    <useLoader/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});