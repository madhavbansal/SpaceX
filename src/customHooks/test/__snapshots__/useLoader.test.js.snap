// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the useLoader Component 1`] = `<useLoader />`;
